lib/services/product_brutto_price.rb

```diff
@@ -0,0 +1,8 @@
+module Store
+  class ProductBruttoPrice
+    def call(basket_id, product_id)
+      product = FetchProductFromBasket.new.call(basket_id, product_id)
+      product.quantity * product.price * (1 + product.vat)
```

Comment:
Not sure what you mean. My suggestion was to stay consequent with spaces between mathematic symbols. Either you separate symbols with space or you don't do it at all